packages/@unimodules/react-native-adapter/src/errors/UnavailabilityError.ts

```diff
@@ -1,4 +1,5 @@
-import { Platform } from '../index';
+import { Platform } from 'react-native';
```

Comment:
instead of importing from react-native, which changes the semantics, split platform out into a separate module and import it from both locations to break the cycle

Reply:
@ide done ✔️